### PR TITLE
added plant sitter modal and functionality.. just need to add plants …

### PR DESCRIPTION
**Pull request by skrantzz**
…you are sitting to the page

### CODE REVIEW

File: client/src/pages/Plantkins.js [hunk 1]
```diff
@@ -61,7 +88,10 @@ function Plants() {
         setUser(res.data[0]);
       })
       .catch((err) => console.log(err));
-  }, [user]);
+
```
Comment (mlfiii) [inline client/src/pages/Plantkins.js:91]: On line 64, please add user back in.  This is what actually forces a refresh of the page after the favorite button is clicked.

File: client/src/pages/Plantkins.js [hunk 2]
```diff
@@ -61,7 +88,10 @@ function Plants() {
         setUser(res.data[0]);
       })
       .catch((err) => console.log(err));
-  }, [user]);
+    
+      getUsers().then(({data})=> setUserList(data.filter(a=> !(a.user_id === userId))))
+    
+  }, []);
```
Comment (VartanyanE) [inline client/src/pages/Plantkins.js:94]: just add [user] back into the dependency array

Comment (mlfiii) [reply client/src/pages/Plantkins.js:94]: How would I see that it is actually there.  Also, I would take out the console.logs on lines 73 and 56.

Comment (mlfiii) [reply client/src/pages/Plantkins.js:94]: There are now some conflicts.  I believe they were caused when master was last merged.

**Pull request merged**